### ISSUE TITLE
Could you Please Provide COPR REPO for Fedora 34?

### ISSUE THREAD

**Issue by hhlp** [jhaygood86/fedora-copr-pop-os-gtk-theme#1]
pop-os-gtk-theme

This is a great gtk them for fedora I foudn it and is very nice could you provide the package for fedora 34??? in your COPR repo.

and thx for your work....

Regards.,

**Comment by R0b0ty** [comment]
I totally agree with @hhlp and would love to be able to use that theme on Fedora 34.  **In fact**... believe it or not I haven't upgraded to Fedora 34 JUST because I'd lose out on this theme.  I'm so used to it!
